src/Tickets/Seating/Tables/Layouts.php

```diff
@@ -87,6 +87,7 @@ protected function get_definition() {
 				`created_date` datetime NOT NULL,
 				`map` varchar(36) NOT NULL,
 				`seats` int(11) NOT NULL DEFAULT '0',
+				`screenshot_url` varchar(255) NOT NULL,
```

Comment:
Set a default of empty string?
The value might legitimately be an empty string, let's account for that.